src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+}
```

Comment:
The BEM block `.stars` should not have any elements or modifiers defined within its block. Each BEM element or modifier should be defined in a new root level of the SCSS file. This helps to keep the stylesheet more maintainable and easier to understand.